app/screens/channel/channel_nav_bar/index.tsx

```diff
@@ -0,0 +1,117 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+import React, {useEffect, useState} from 'react';
+import {DeviceEventEmitter, LayoutChangeEvent, Platform, View} from 'react-native';
+import DeviceInfo from 'react-native-device-info';
+import {useSafeAreaInsets} from 'react-native-safe-area-context';
+
+import {useTheme} from '@context/theme';
+import VIEWS from '@constants/view';
+import DEVICE from '@constants/device';
+import {General} from '@constants';
+import {getUserIdFromChannelName} from '@utils/user';
+import {makeStyleSheetFromTheme} from '@utils/theme';
+
+import ChannelTitle from '../channel_title';
+
+import type ChannelModel from '@typings/database/models/servers/channel';
+
+type ChannelNavBar = {
+    channel: ChannelModel;
+    currentUserId: string;
+    onPress: () => void;
+    config: ClientConfig;
+}
+
+const ChannelNavBar = ({currentUserId, channel, onPress, config}: ChannelNavBar) => {
+    const insets = useSafeAreaInsets();
+    const theme = useTheme();
+    const style = getStyleFromTheme(theme);
+    let height = 0;
+    let canHaveSubtitle = true;
+
+    const [isLandscape, setIsLandscape] = useState<boolean>(false);
```

Comment:
The default value should be `Dimensions.width > Dimensions.height`

Reply:
I created a new constant under `@constant/device` that we'll reference for this.